src/phaser/components/minigame/simon-says-board.ts

```diff
@@ -1,4 +1,4 @@
-import { Audio } from "../types";
+import { Audio } from "../../../types";
```

Comment:
Very inconsistent in some files @ is uses and in some not, is there a reason for that?